doc/api/debugger.md

```diff
@@ -100,6 +100,16 @@ To begin watching an expression, type `watch('my_expression')`. The command
 `watchers` will print the active watchers. To remove a watcher, type
 `unwatch('my_expression')`.
 
+## Options
+
+### `--port=port`
```

Comment:
Should we also add comments that specify when the features were added? Take a look at `cli.md`: https://github.com/nodejs/node/blob/c4781ea69ce164b1163a10f50d818be3d0a6bca4/doc/api/cli.md?plain=1#L92